### PR TITLE
updated blink C/C++ samples.

### PR DESCRIPTION
**Pull request by shahabl**
Not for merge.

### CODE REVIEW

**Comment by anjgola** [comment]
LGTM!

File: onboard-blink/c/blink.c [hunk 1]
```diff
@@ -22,31 +22,66 @@
...
+		case MRAA_UP2:
+#ifdef USING_GROVE_PI_SHIELD
+			gpioPin = 4 + 512; // D4 Connector (512 offset needed for the shield)
+			break;
+#endif
+		case MRAA_UP:
+		case MRAA_INTEL_EDISON_FAB_C:
+		case MRAA_INTEL_GALILEO_GEN2:
+			break;
+		case MRAA_INTEL_MINNOWBOARD_MAX: // Same for Minnowboard Turbut
```
Comment (anjgola) [inline onboard-blink/c/blink.c]: *Turbot

File: onboard-blink/c/blink.c [hunk 2]
```diff
@@ -22,31 +22,63 @@
...
- /**
-  * TODO use a platform with GPIO capabilities and an onboard LED
-	*/
+/* On-board LED Blink
+ * Turns the on-board LED on for one second, then off for one second, repeatedly.
+ */
 
 #include <mraa.h>
-
+#include <mraa/common.h>
```
[the remark this answers is not in the document]
Comment (shahabl) [reply onboard-blink/c/blink.c]: It's actually not needed now, I'll remove it.

File: onboard-blink/c/blink.c (hunk 1 above)
Comment (shahabl) [reply onboard-blink/c/blink.c]: Thanks, I'll fix it!

Comment (anjgola) [reply onboard-blink/c/blink.c]: Turbo turbo turbo......testing!

File: bluemix-quickstart/cpp/main.cpp [hunk 3]
```diff
@@ -43,7 +43,7 @@ extern "C" {
     
     const char * topic = "iot-2/evt/status/fmt/json";
     const char * clientID = strcat(
-            strcpy(new char[50], "d:quickstart:iotquick-edison:"), DEVICE_ID);
+            strcpy(new char[50], "d:quickstart:iotquick-my-iot-board:"), DEVICE_ID);
```
Comment (anjgola) [inline bluemix-quickstart/cpp/main.cpp]: This sample still has the old reference board check. Is that for next week?

File: onboard-blink/maven/src/main/java/iotdk/example/OnboardLedBlink.java [hunk 4]
```diff
@@ -41,7 +41,7 @@
 public class OnboardLedBlink {
 
     public static void main(String[] args) {
-        //TODO Change the GPIO to one that matches your platform
+        // Change the GPIO to one that matches your platform
         // Intel Galileo Gen 2 = 13
```
Comment (anjgola) [inline onboard-blink/maven/src/main/java/iotdk/example/OnboardLedBlink.java]: Needs upsquare pin.

**Comment by anjgola** [comment]
LGTM Shahab. I will test these today.

File: bluemix-quickstart/cpp/main.cpp (hunk 3 above)
Comment (shahabl) [reply bluemix-quickstart/cpp/main.cpp]: yes

**Pull request merged**